docs/dapps/developer-toolchains/README.md

```diff
@@ -0,0 +1,9 @@
+# Developer Toolchains
```

Comment:
I don't see how this can be navigated to as https://standardize-dapp-guides.avalanche-docs.pages.dev/dapps/developer-toolchains. A new entry in the sidebar.js should be added so that it shows up on the left bar.

Reply:
Also the siderbar.js needs to be re-factored to follow the new structure and make all docs showing up.

Reply:
I refactored `sidebar.js` to add a developer toolchains section so that it's README is accessible. Also sorted the sidebar sections alphabetically.

Are there are other sidebar refactors which you're referring to? If so can you point me to an example in the code so I can have a reference for the new structure you're mentioning? Thx

Reply:
You got it all. No other sidebar refactor needed.